openshift/template.yaml

```
@@ -71,6 +124,16 @@ parameters:
 - name: IMAGE_PULL_POLICY
   value: "Always"
 - name: SUSPEND
-  value: "false"
+  value: "true"
 - name: SCHEDULE
   value: "@hourly"
+- name: JOBS_AUTO_REPORT_LOG_LEVEL
+  value: "INFO"
+- name: JOBS_AUTO_REPORT_SUSPEND
+  value: "false"
```

```suggestion
  value: "true"
```
it's the jobs-auto-report we want to suspend by default